When testing usage, the following error occurs:
```
MSBuild version 17.11.2+c078802d4 for .NET Framework

  test.cpp
E:\graaf\installed\x64-windows\include\graaf\graaflib\graph.h(3,10): error C1083: Cannot open include file: 'graaflib/edge.h': No such file or directory [E:\test_usage\test\build\m
ain.vcxproj]
  (compiling source file '../test.cpp')
```

<details>
<summary><b>test.cpp</b></summary>
<pre>
#include &#60;iostream&#62;
#include "graaf/graaflib/graph.h"

using namespace std;

int main()
{
        cout << "Hello CMake." << endl;
        return 0;
}
</pre></details>


<details>
<summary><b>CMakeLists.txt</b></summary>
<pre>
cmake_minimum_required (VERSION 3.8)

set(CMAKE_TOOLCHAIN_FILE "E:/graaf/scripts/buildsystems/vcpkg.cmake")

project ("test")

add_library (main "test.cpp")

find_path(GRAAF_INCLUDE_DIRS "graaf/graaflib/graph.h")
target_include_directories(main PRIVATE ${GRAAF_INCLUDE_DIRS})
</pre></details>

--- a/ports/graaf/portfile.cmake
+++ b/ports/graaf/portfile.cmake
@@ -1,0 +1,11 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO bobluppes/graaf
+    REF "v${VERSION}"
+    SHA512 e97eeadaab079cf6ff429d2580ea7be454a6583b8cc5cd231065c7c51a87d52d60457370dc9688a7e426ffc7ef79ad9670e44966dd367224a4124bcd5755f080
+)
+
+file(COPY "${SOURCE_PATH}/include/graaflib" DESTINATION ${CURRENT_PACKAGES_DIR}/include)
+
+file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE.md")

--- a/ports/graaf/usage
+++ b/ports/graaf/usage
@@ -1,0 +1,4 @@
+graaf is header-only and can be used from CMake via:
+
+  find_path(GRAAF_INCLUDE_DIRS "graaflib/graph.h")
+  target_include_directories(main PRIVATE ${GRAAF_INCLUDE_DIRS})

--- a/ports/graaf/vcpkg.json
+++ b/ports/graaf/vcpkg.json
@@ -1,0 +1,7 @@
+{
+  "name": "graaf",
+  "version": "1.1.1",
+  "description": "A general-purpose lightweight C++ graph library",
+  "homepage": "https://bobluppes.github.io/graaf/",
+  "license": "MIT"
+}

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3252,6 +3252,10 @@
       "baseline": "2020-05-20",
       "port-version": 4
     },
+    "graaf": {
+      "baseline": "1.1.1",
+      "port-version": 0
+    },
     "grantlee": {
       "baseline": "5.3.1",
       "port-version": 2

--- a/versions/g-/graaf.json
+++ b/versions/g-/graaf.json
@@ -1,0 +1,9 @@
+{
+  "versions": [
+    {
+      "git-tree": "608792629708669aef95575491d7ff4d009fa548",
+      "version": "1.1.1",
+      "port-version": 0
+    }
+  ]
+}

